File: Bai_46/main.cpp
#include <iostream>
#include <unordered_map>
#include <vector>
using namespace std;
int main() {
    vector<int> numbers;
    int num;

    // Nhập danh sách các số
    cout << "Nhap danh sach cac so (cach nhau bang dau cach): ";
    while (cin >> num) {
        numbers.push_back(num);
    }

    // Đếm số lần xuất hiện của mỗi số
    unordered_map<int, int> count;
    for (int num : numbers) {
        if (count.find(num) != count.end()) {
            count[num]++;
        } else {
            count[num] = 1;
        }
    }

    // In số lần xuất hiện của mỗi số
    for (const auto& pair : count) {
        cout << "Số " << pair.first << " lặp lại " << pair.second << " lần\n";
    }

    return 0;
}

File: Bai_46/main.py
numbers = list(map(int, input().split()))
count = {}
for num in numbers:
    if num in count:
        count[num] += 1
    else:
        count[num] = 1
for num in count:
    print(f"Số {num} lặp lại {count[num]} lần")


from collections import Counter
for key,value in Counter(numbers).items():
    print(f'Số {key} lặp lại {value} lần')
